src/calculation/SaleElement.ts

```diff
@@ -0,0 +1,28 @@
+import { Currency } from '~/models/Currency';
+import { Sale } from '~/models/Sale';
+import { VatRate } from '~/models/VatRate';
+
+import { roundSum } from './roundSum';
+
+export class SaleElement implements Sale {
+  public code: string | null = null;
+  public readonly currency: Currency = Currency.PLN;
```

Comment:
Maybe currency could be set in separated (SomeSettings) class and SaleElement and CostElement will inherit it? We won't need to set it in two (for now) places?

Reply:
This might be a good idea. I'll have a look and let you know later.

Reply:
For the reason we it is possible we will have invoices in different currency it is fine